### ISSUE TITLE
Breakpoints not working - "Breakpoint set but not yet bound"

### ISSUE THREAD

**Issue by Rohit459** [microsoft/vscode#122580]
- VS Code Version: 1.55
- OS Version: Windows 10-64
- Framework and tools: TS, MS Bot Framework v4(NodeJS)

Steps to Reproduce:

1. Build the project.
2. Start the debugger and set a break point in the ts file.
3. When it launches all of the break points change to "Breakpoint set but not yet bound" 

Attaching screenshots for reference.
![image](https://user-images.githubusercontent.com/23269340/116466598-b26e2880-a83c-11eb-81fa-47c7f6a76b28.png)

My Launch Config:
![image](https://user-images.githubusercontent.com/23269340/116466840-fcefa500-a83c-11eb-8645-609bd18281a8.png)

sourcemap is set to true in tsConfig.

Does this issue occur when all extensions are disabled?: Yes

**Comment by Rohit459** [comment]
Hi Here is the file that you asked for
[vscode-debugadapter-f73851f1.json.gz](https://github.com/microsoft/vscode/files/6405739/vscode-debugadapter-f73851f1.json.gz)

**Comment by connor4312** [comment]
It looks like you might have symbolically linked sources. If you do, follow the "Can I debug if I'm using symlinks" steps here: https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_can-i-debug-if-im-using-symlinks

**Comment by msalvi96** [comment]
I am facing a similar issue as well where the breakpoints I set before a debug session get transformed into unbound breakpoints at debug runtime and the debugger stops at compiled JS files whereas I want those breakpoints to stop at the initial TS files. Also set the trace=true property in my launch config but the breakpoints still don't stop at the desired point in TS files. 

VS Code - 1.55
OS - Windows 10 64
Language - TypeScript 4.0
Framework - Azure Bot Framework SDK

Breakpoint Screenshot (Before starting debug session)
![set-breakpoint](https://user-images.githubusercontent.com/48700772/116878086-d0e17480-abec-11eb-8881-52b2eec7edd7.PNG)
![breakpoint-list](https://user-images.githubusercontent.com/48700772/116878107-db037300-abec-11eb-81fe-6d272eef04ca.PNG)

Breakpoint Screenshot (After starting debug session)
![unbound-breakpoint](https://user-images.githubusercontent.com/48700772/116878153-efe00680-abec-11eb-9e29-924b1762faa6.PNG)
![unbound-list](https://user-images.githubusercontent.com/48700772/116878164-f40c2400-abec-11eb-9c6a-eac574e68fe5.PNG)

Breaks at JS Files
![js-break](https://user-images.githubusercontent.com/48700772/116878207-05553080-abed-11eb-9b38-9ba541a8f528.PNG)

Launch Configuration
![launch config](https://user-images.githubusercontent.com/48700772/116878329-30d81b00-abed-11eb-9d0c-66e2f7128e06.PNG)

**Comment by connor4312** [comment]
@msalvi96 please open a separate issue and collect a log file using the instructions in https://github.com/microsoft/vscode/issues/122580#issuecomment-829357921

**Comment by boobzzz** [comment]
Facing the same issue. When putting BP in my JS files debug works fine, with TS files BP become unbound.
Here is the log file:
[vscode-debugadapter-ba4013ef.json.gz](https://github.com/microsoft/vscode/files/6425863/vscode-debugadapter-ba4013ef.json.gz)

**Comment by connor4312** [comment]
@boobzzz you have a different issue. Your code is not being loaded or does not have sourcemaps.
